### DISCUSSION TITLE
milvus-standalone服务 自动退出

### DISCUSSION THREAD

**Discussion by Qyijiu** [milvus-io/milvus#16141] [Q&A and General discussion]
![image](https://user-images.githubusercontent.com/90362695/159445299-2f29fec9-986d-4ef5-81b7-c8a2a200858d.png)
docker logs milvus-standalone 输出的命令结果如上

**Comment by xiaofan-luan** [comment]
please just wait for 2.0.2 release or use the day build.

**Reply by Qyijiu** [reply]
What is use on the day build，How can I solve or avoid this situation

**Reply by Qyijiu** [reply]
and I don't know what I did to cause this. I want to avoid it. = =、
